.gitignore

```diff
@@ -1,79 +0,0 @@
-.history/
```

Comment:
Please revert the changes to this file.

Reply:
I reverted the file to what it was in the master, hopefully it's correct now